include/solvers/slepc_eigen_solver.h

```diff
@@ -267,6 +267,13 @@ class SlepcEigenSolver : public EigenSolver<T>
    */
   EPS _eps;
 
+  /**
+   * Store a set of \p _initial_space_n initial
+   * guesses in \p _initial_space_vect.
+   */
+  int _initial_space_n;
```

Comment:
This chunk should probably no longer be part of the diff since you are going to use @juan42's interface for that instead?